scripts/jobs/cron_tasks.inc.http.20.lighttpd.php

```diff
@@ -435,11 +435,13 @@ protected function getVhostContent($domain, $ssl_vhost = false, $ipid)
 
 			// Get domain's redirect code
 			$code = getDomainRedirectCode($domain['id'], '301');
-
-			$vhost_content .= '  url.redirect-code = ' . $code. "\n";
-			$vhost_content .= '  url.redirect = (' . "\n";
-			$vhost_content .= '     "^/(.*)$" => "' . $uri . '$1"' . "\n";
-			$vhost_content .= '  )' . "\n";
+            
+            $vhost_content .= '  $HTTP["host"] =~ "^(.*)$" {'. "\n";
```

Comment:
This code use %1/$1 with brackets in match string: http://superuser.com/questions/317075/redirect-from-http-to-https-in-lighthttpd/319357#319357
:confused: